.github/workflows/create_release.yml

```
@@ -36,6 +36,7 @@ jobs:
         run: |
             # Create new folder with specified name so extracting the archive yields that
             rm -rf "/tmp/$PT_RELEASE_NAME"
+            mkdir "/tmp/$PT_RELEASE_NAME"
```

Shouldn't it be `-p`?
```suggestion
            mkdir -p "/tmp/$PT_RELEASE_NAME"
```

---

Used suggested approach: `` tag_or_branch="${tag_or_branch//\//_}"``